src/components/shared/CreateButton/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './createButton.module.scss';
+
+const CreateButton = ({
+  onclick, classes, children, type
+}) => (
+  <button type={type} onClick={onclick} className={[styles.buttonCommon, ...classes].join(' ')}>
```

Comment:
We are using classnames lib already
So you can write it as classNames(styles.buttonCommon, classes) and not use join